components/tidb_query_expr/src/impl_string.rs

```diff
@@ -3218,6 +3226,54 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_instr() {
+        let cases = vec![
+            (Some(b"a".to_vec()), Some(b"abcdefg".to_vec()), Some(1)),
+            (Some(b"0".to_vec()), Some(b"abcdefg".to_vec()), Some(0)),
+            (Some(b"c".to_vec()), Some(b"abcdefg".to_vec()), Some(3)),
+            (Some(b"F".to_vec()), Some(b"abcdefg".to_vec()), Some(0)),
+            (Some(b"cd".to_vec()), Some(b"abcdefg".to_vec()), Some(3)),
+            (Some(b" ".to_vec()), Some(b"abcdefg".to_vec()), Some(0)),
+            (Some(b"".to_vec()), Some(b"".to_vec()), Some(1)),
+            (Some(b" ".to_vec()), Some(b"".to_vec()), Some(0)),
+            (Some(b"".to_vec()), Some(b" ".to_vec()), Some(1)),
+            (Some(b"eFg".to_vec()), Some(b"abcdefg".to_vec()), Some(0)),
+            (Some(b"deF".to_vec()), Some(b"abcdefg".to_vec()), Some(0)),
+            (
+                Some("字节".as_bytes().to_vec()),
+                Some("a多字节".as_bytes().to_vec()),
+                Some(1 + "a多字节".find("字节").unwrap() as i64),
```

Comment:
Please give the actual answer instead fo a call to `find`.

Reply:
done

PTAL